marlowe-dashboard-server/src/Marlowe/Run/Webserver/Wallet/CentralizedTestnet/Server.hs

```diff
@@ -0,0 +1,145 @@
+{-# LANGUAGE DataKinds             #-}
```

Comment:
The logic on this file hasn't change basically... only two minor changes commented below